### DISCUSSION TITLE
c# programming in this application

### DISCUSSION THREAD

**Discussion by Don1L** [thebaselab/codeapp#537] [Q&A] [answered]
Hello.  I want to buy this application but I have a couple of questions.  Is it possible to program normally in c# in this application and will it be possible to use the code written here to work later with it in Unity

**Comment by bummoblizard** [comment, marked as answer]
Hi. Support for C# is limited to syntax highlighting and online execution. You can only run simple script with no external dependencies and it requires an internet connection.

You can try it on TestFlight first: https://testflight.apple.com/join/EgZ8sE2P. Online execution is unavailable on TestFlight though.

**Reply by Don1L** [reply]
that is, I can use the standard libraries and write not too complicated codes without problems?

**Reply by bummoblizard** [reply]
Yes. We use judge0 as the code execution backend so you can try it here first to confirm: https://ide.judge0.com

**Reply by Don1L** [reply]
thank you very much i think this is what i need

**Reply by Don1L** [reply]
and I have one more question, is it possible to save projects with the .cs extension in the full version of the program and open them on a computer in vs, just in the test version, when you save the project and open it in vs, it is completely empty
